[Marketplace Contribution] Common Scripts - Content Pack Update (#33152)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_15_0.md

#### Scripts

##### CreateNewIndicatorsOnly

> - Added functionality to associate the passed in indicators with the incident 

Reviewer: ```suggestion
- Added the functionality to associate an indicators with an incident.
```

- Updated the Docker image to: *demisto/python3:3.10.13.87159*.

> ##### New: PCAPMiner
> 
> - New: Deprecated. Use PCAPMinerV2 instead. PCAPMiner is a tool to parse PCAP files and will return things like extracted files that are found, HTTP flows, and a variety of other information. It is uses a docker instance located on docker hub trorabaugh/dempcap:1.0.  To use simply upload a PCAP file and then run PCAPMiner entryId="<your_entry_id>". To get the entry id click on the link on the top right hand corner of a file attachment. (Available from Cortex XSOAR 6.8.0).
> ##### New: BinarySearchPy
> 
> - New: Deprecated. No available replacement. Search for a binary on an endpoint using Carbon Black (Available from Cortex XSOAR 6.8.0).

Reviewer: ```suggestion
```
